crates/polars-plan/src/dsl/function_expr/temporal.rs

```diff
@@ -6,6 +6,100 @@ use polars_time::prelude::*;
 
 use super::*;
 
+#[cfg(feature = "timezones")]
```

Comment:
I'm not happy with the feature gate, but it's required to use `replace_time_zone`. Is there any way around this?

Reply:
a pattern like this is used elsewhere in the code, perhaps that works here?

https://github.com/pola-rs/polars/blob/128803b237dc13d0522c22dbccae1257ae30477e/crates/polars-core/src/chunked_array/cast.rs#L45-L52

Reply:
Thanks, I'll look into this!